yarn.lock

```diff
@@ -9,6 +9,46 @@
   dependencies:
     "@babel/highlight" "^7.0.0"
 
+"@babel/code-frame@^7.8.3":
```

Comment:
Why are all these changes here? They are unrelated to your fix...

Reply:
The formatting tests were failing due to the outdated yarn.lock file. So I updated it.
Do you think this should be a separate PR?

Reply:
Yep. PRs should be focused on one thing.

Reply:
Fixed!